packages/client/src/topics.json

```diff
@@ -19,8 +19,8 @@
       "path": "imtr/transformed/bouwwerk-slopen.json",
       "permits": [
         "bxBCdnrFZSbwxgmxC",
-        "2eYY7McaiRgfnERo9",
-        "D4mccynMtNdzYGhXF"
+        "D4mccynMtNdzYGhXF",
+        "2eYY7McaiRgfnERo9"
```

Comment:
This order doesn't matter I think (it's generated anyway), but it's nice to see it here anyway.